org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/queues/beans/ScanAtom.java

```diff
@@ -127,7 +127,7 @@ public ScanAtom(String scShrtNm, Map<String, DeviceModel> pathModels, Map<String
 		detectorModelsModel = detectorModels;
 		monitorsModel = monitors;
 	}
-	
+
 	public ScanRequest<?> getScanReq() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)